packages/handlers/openapi/src/index.ts

```diff
@@ -1,24 +1,67 @@
-import { readFileOrUrlWithCache } from '@graphql-mesh/utils';
-import { createGraphQLSchema } from 'openapi-to-graphql';
-import { Oas3 } from 'openapi-to-graphql/lib/types/oas3';
+import {
+  readFileOrUrlWithCache,
+  ResolverData,
+  parseInterpolationStrings,
+  stringInterpolator,
+} from '@graphql-mesh/utils';
+import { createGraphQLSchema } from '@ardatan/openapi-to-graphql';
```

Comment:
Please open a PR with your changes to the original repo, they usually respond quickly.